includes/polling/os/kti.inc.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
+$hardware     = trim(snmp_get($poll_device['sysDescr'], ".1.3.6.1.2.1.1.1.0", '-Ovq'), '"');
```

Comment:
What murrant meant here was that $poll_device['sysDescr'] already contains the output you want so you don't need another snmp call just $hardware = $poll_device['sysDescr']